### ISSUE TITLE
Swipeable ListItem slide action

### ISSUE THREAD

**Issue by jorgegvallejo** [react-native-elements/react-native-elements#3676]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Explain what you did

Using the ListItem Swipeable component with a left and a right button behind it.

### Expected behavior

Should be able to slide the ListItem smoothly and easily.

### Describe the bug

When sliding the ListItem either left or right, the sliding action works well as long as your finger only slides horizontally. If you slide the item and your finger slides with any vertical effect/angle, the slide action is lost and the list item resets to its original position. This makes it very difficult to slide.

In other words, it seems like the ListItem only slides if you slide perfectly horizontally. Swipeable ListItems in other apps don't do this - they allow swipe motions with a vertical angle to them.

### Steps To Reproduce

```Markdown
Swipe a ListItem to either the left or right, but as it starts sliding, slide your finger at a vertical angle instead of perfectly horizontal.
```

### Your Environment

RN 0.70.3
RNEUI 4.0.0-rc.4
